corehq/apps/case_importer/do_import.py

```diff
@@ -322,8 +326,8 @@ def _populate_updated_fields(self, row):
             elif update_value is not None:
                 update_value = _convert_field_value(update_value)
 
-            if update_field_name in self.fields_to_validate:
-                case_property = self.fields_to_validate[update_field_name]
+            if self.field_to_case_property:
+                case_property = self.field_to_case_property[update_field_name]
```

Comment:
This doesn't seem to be a direct translation of the previous statement. The previous version was checking that the key was present in `fields_to_validate` before accessing it. The changed version appears to introduce the possibility that accessing `update_field_name` will generate a `KeyError`. Is this something we need to handle?